libsignal/src/main/java/org/session/libsignal/crypto/MnemonicCodec.kt

```diff
@@ -57,10 +57,10 @@ class MnemonicCodec(private val loadFileContents: (String) -> String) {
         object VerificationFailed : DecodingError("Your mnemonic couldn't be verified. Please check what you entered and try again.")
     }
 
-    fun encode(hexEncodedString: String, languageConfiguration: Language.Configuration = Language.Configuration.english): String {
+    suspend fun encode(hexEncodedString: String, languageConfiguration: Language.Configuration = Language.Configuration.english): String {
```

Comment:
once we've loaded the file should we swap to `computation` for encode and decode?